h/security/policy/_api_cookie.py

```diff
@@ -29,7 +29,7 @@ def handles(request: Request) -> bool:
         ) in COOKIE_AUTHENTICATABLE_API_REQUESTS
 
     def identity(self, request: Request) -> Identity | None:
-        identity = self.helper.identity(self.cookie, request)
+        identity, _ = self.helper.identity(self.cookie, request)
```

Comment:
`self.helper.identity()` now returns an `(Identity, AuthTicket)` 2-tuple. `APICookiePolicy` doesn't need the new `AuthTicket` part of the return value so just ignores it.